src/components/Pages/LoginPage/LoginPage.tsx

```diff
@@ -33,7 +33,7 @@ export const LoginPage = () => {
         if (redirectTo) {
           window.location.href = redirectTo
         } else {
-          navigate(navigateTo ?? '/user')
+          // Go to explorer
```

Comment:
What should be the behavior here? There are cases where you don't get redirected?

Reply:
I wasn't sure whether to change the structure and now that we don't have the /user delete the else. Or if we should do something in the case that we should not redirect, I don't know if there is such a case.